docs/book/codegen.md

```diff
@@ -84,6 +87,7 @@ return [
             'aot' => [
                 'namespace' => 'AppAoT\Generated',
                 'directory' => __DIR__ . '/../gen',
+                'logger' => \Psr\Log\LoggerInterface::class,
```

Comment:
The same here: the leading backslash for the classname can be omitted because it is only needed in namespaced code.